src/angular-app/bellows/main.common.ts

```diff
@@ -5,7 +5,7 @@ import 'angular-ui-router';
 import 'angular-ui-validate';
 import 'angular-zxcvbn';
 
-import 'ng-file-upload/dist/ng-file-upload-all.js';
+import 'ng-file-upload/dist/ng-file-upload-all.min.js';
```

Comment:
Same comment as for the `angular-moment-picker` import. Why are we importing a specific file from a package? That doesn't seem right.